packages/trader/src/Modules/MT5/Components/mt5-poi.jsx

```diff
@@ -0,0 +1,98 @@
+import { Formik }               from 'formik';
+import PropTypes                from 'prop-types';
+import React, { PureComponent } from 'react';
+import {
+    ThemedScrollbars,
+    FormSubmitButton }          from '@deriv/components';
+import { localize }             from '@deriv/translations';
+import ProofOfIdentityContainer from 'Modules/Account/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx';
+
+class MT5POI extends PureComponent {
+    state = {
+        poi_state: 'none',
+    };
+
+    onStateChange = ({ status }) => this.setState({
+        poi_state: status,
+    });;
+
+    validateForm = () => {
+        const errors = {};
+        if (!['pending', 'verified'].includes(this.state.poi_state)) {
+            errors.poi_state = localize('You need a successful/pending POI status before continue.');
```

Comment:
This text seems grammatically incorrect. Can we review this?

Reply:
I have checked, we aren't going to show this error at all. I've removed it and replaced it with a boolean.